docs/audit-log-filter-json.md

```diff
@@ -0,0 +1,92 @@
+# Audit Log Filter format - JSON
+
+The feature is in [tech preview](glossary.md#tech-preview).
+
+The JSON format uses JSON arrays that contain properties as a key-value pair. These arrays represent an event in the audit. Some pairs are listed in every audit record. The audit record type determines if other key-value pairs are listed. The order of the pairs within an audit record is not guaranteed. The value description may be truncated.
+
+Certain statistics, such as query time and size, are only available in the JSON format and help detect activity outliers when analyzed. 
+
+```json
+
+[
+  {
+    "timestamp": "2023-03-29 11:17:03",
+    "id": 0,
+    "class": "audit",
+    "server_id": 1
+  },
+  {
+    "timestamp": "2023-03-29 11:17:05",
+    "id": 5,
```

Comment:
Please update "id" for each object to make them monotonically growing for each event starting from 0.